addons/skin.confluence/720p/DialogBusy.xml

```diff
@@ -8,6 +8,7 @@
 	</coordinates>
 	<controls>
 		<control type="group">
+			<depth>DepthDialogue+</depth>
```

Comment:
as said in another commit already, "Dialog" instead of "Dialogue" would be more consistent with general skinning labelling.